httpcore5/src/main/java/org/apache/hc/core5/reactor/IOReactorConfig.java

```diff
@@ -464,7 +464,7 @@ public IOReactorConfig build() {
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
-        builder.append("[selectIntervalMillis=").append(this.selectInterval)
+        builder.append("[selectInterval=").append(this.selectInterval)
```

Comment:
I can't imagine why would one would want to hide the scale.

Reply:
Because you don't remember your code: https://github.com/apache/httpcomponents-core/blob/9c05aafcde239e508d67a15716104773749d9f7e/httpcore5/src/main/java/org/apache/hc/core5/util/TimeValue.java#L377-L380 ;-)

Reply:
I don't remember what I had for breakfast! ;-)